apps/gql/package.json

```
@@ -29,6 +29,7 @@
   "dependencies": {
     "@graphql-tools/schema": "9.0.19",
     "@kampus/gql-utils": "*",
+    "@kampus/next-auth": "^0.0.0",
```

```suggestion
    "@kampus/next-auth": "*",
```
any reason why gql requires auth ?